packages/types/src/interfaces/contractsAbi/types.ts

```diff
@@ -161,6 +170,15 @@ export interface ContractEventSpecV2 extends Struct {
   readonly docs: Vec<Text>;
 }
 
+/** @name ContractEventSpecV3 */
+export interface ContractEventSpecV3 extends Struct {
+  readonly label: Text;
+  readonly args: Vec<ContractEventParamSpecV2>;
+  readonly docs: Vec<Text>;
+  readonly module_path: Text;
+  readonly signature_topic: U8aFixed;
+}
```

Comment:
Not sure if these are the correct types to be used here. 🤔 
`signature_topic` being a hash and `module_path` a `::` path.

Reply:
In this case the API's types for `Hash` end up resolving to a U8aFixed: https://github.com/polkadot-js/api/blob/master/packages/types/src/interfaces/runtime/types.ts#L158-L170

So I would imagine in this case `signature_topic` being set as `U8aFixed` is correct.